resources/views/settings/labels.blade.php

```diff
@@ -179,6 +179,10 @@
                                         {{ Form::checkbox('labels_display_tag', '1', Input::old('labels_display_tag',   $setting->labels_display_tag),array('class' => 'minimal')) }}
                                         {{ trans('admin/hardware/form.tag') }}
                                     </label>
+                                    <label>
+                                        {{ Form::checkbox('labels_display_company_name', '1', Input::old('labels_display_company_name',   $setting->labels_display_company_name),array('class' => 'minimal')) }}
```

Comment:
This is nitpicky, but can you use the square brace array notation, just to keep it consistent with other blades? 

```
 {{ Form::checkbox('labels_display_company_name', '1', Input::old('labels_display_company_name',   $setting->labels_display_company_name),['class' => 'minimal']) }}
```